lib/ex_unit/lib/ex_unit/diff.ex

```diff
@@ -66,6 +71,18 @@ defmodule ExUnit.Diff do
 
   def format(_left, _right, _formatter), do: nil
 
+  defp proceed_string_difference?(left, right) do
+    sum_length = String.length(left) + String.length(right)
```

Comment:
I initially thought about `binary_size` but given that `jaro_distance` and `myers_difference` work on grapheme level it could be a bit unpredictable.
Also `String.length` execution time is unnoticeable in comparison with `myers_difference`.
Probably we can have a guard for 10000 bytes size of a strings to be always on a safe side.